### ISSUE TITLE
Wrong Favicon's ALT text in BO

### ISSUE THREAD

**Issue by Caleydon** [PrestaShop/PrestaShop#19131]
#### Describe the bug

Wrong Favicon's ALT text in BO (Design > Theme & Logo). Nothing crucial ;)

#### Steps to Reproduce

1. Go to Design > Theme & Logo
2. Right-click on favicon image in the upload part and see the wrong ALT text "Invoice logo". It should be "Favicon image" or something like this.

See the video: https://vimeo.com/418440686/e5af815c73

#### Additional information

* PrestaShop version: 1.7.7 beta 1 (but also in previous 1.7 versions)
* PHP version: 7.2

**Comment by florine2623** [comment]
Hi @Caleydon ,
Thanks for the report, please stay tuned !

**Comment by Klemart3D** [comment]
Suggested PR: #19507
